<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InteShelf</class>
 <widget class="QWidget" name="InteShelf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="bindingTW">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>40</y>
     <width>168</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="huajiaTW">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>168</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="mainpageTW">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>40</y>
     <width>168</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>755</width>
     <height>314</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>755</width>
       <height>314</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="eg1nameLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="eg1cardLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>95</y>
       <width>180</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="eg2nameLabel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="eg2cardLabel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>95</y>
       <width>180</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="eg3nameLabel">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="eg3cardLabel">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>95</y>
       <width>180</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="eg4nameLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>185</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="eg4cardLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>230</y>
       <width>180</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="eg5nameLabel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>185</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="eg5cardLabel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>230</y>
       <width>180</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="eg6nameLabel">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>185</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="eg6cardLabel">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>230</y>
       <width>180</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>191</width>
       <height>35</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>300</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/0012.png);</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>20</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>120</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Abyssinica SIL</family>
       <pointsize>72</pointsize>
      </font>
     </property>
     <property name="text">
      <string>卡号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Abyssinica SIL</family>
       <pointsize>72</pointsize>
      </font>
     </property>
     <property name="text">
      <string>货物名称</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>191</width>
       <height>35</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_3">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>230</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
